environment.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python=3.11
+  - python=3.11.*
```

Comment:
I don't think this should be necessary? 3.11 should be enough?

Reply:
Cannot create a new env from it on my Mac with only 3.11